typescript/infra/config/environments/test/chains.ts

```diff
@@ -8,3 +10,13 @@ export const testConfigs: ChainMap<ChainMetadata> = {
 
 export type TestChains = keyof typeof testConfigs;
 export const chainNames = Object.keys(testConfigs) as TestChains[];
+
+const validatorChainNames = [...chainNames, 'solanadevnet', 'proteustestnet'];
```

Comment:
as this is just the `test` environment this should stay `const validatorChainNames = chainNames;`